### PR TITLE
[REVIEW] MG PPR

### LINKED ISSUE #996: [ENH] OPG -> MG renaming
OPG is an internal terminology that is not relevant anymore. We should use `Multi-GPU` (`MG`).

- [x] Code : There are many MG PRs that should be merged in the next few days. I suggest we wait for them to be merged (ie. let more `opg` occurrences go in) and do a massive renaming right after so that we have less merge conflicts to resolve.
- [x] Doc :  should be addressed along with the code part
- [x] Gtihub : labels and open issues/PR should be renamed

### LINKED ISSUE #1000: [FEA] Add MG personalized pagerank
We need to upgrade the Pagerank feature so that users can provide some subset of graph vertices and personalization value for each of those.

This information is used to modify the weights of the Google matrix (ie. the vectors used to form it). The power method should work the same after the initial input is altered to reflect the personalization.

This feature involves:
- Upgrading the cuda pagerank solver
- Upgrading python bindings
- Upgrading doc
- Testing
- References

Nx implementation (it is also power iteration based): https://github.com/networkx/networkx/blob/master/networkx/algorithms/link_analysis/pagerank_alg.py

### PR DESCRIPTION
**Pull request by afender**
Closes #1000 
Close #996

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

File: cpp/src/link_analysis/pagerank.cu [hunk 1]
```diff
@@ -355,15 +355,19 @@ void pagerank(raft::handle_t const &handle,
...
-    CUGRAPH_EXPECTS(personalization_subset == nullptr,
-                    "Invalid API parameter: Multi-GPU Pagerank does not support Personalized "
-                    "variant currently, please use the single GPU version for this feature. The "
-                    "multi-GPU support is comming.");
     CUGRAPH_EXPECTS(has_guess == false,
                     "Invalid API parameter: Multi-GPU Pagerank does not guess, please use the "
                     "single GPU version for this feature");
     CUGRAPH_EXPECTS(max_iter > 0, "The number of iteration must be positive");
-    cugraph::opg::pagerank<VT, ET, WT>(handle, graph, pagerank, alpha, max_iter, tolerance);
+    cugraph::opg::pagerank<VT, ET, WT>(handle,
```
Comment (kaatish) [inline cpp/src/link_analysis/pagerank.cu:362]: Are the opg to mg changes only being done in the documentation and not code? One way to make incremental fix without changing the whole code would be to do this :


```
namespace cugraph
{
namespace opg
{
void my_func(void) {}
}
namespace mg = opg;
}

void calling_func(void) {
  cugraph::mg::my_func();
}

```

File: python/cugraph/opg/link_analysis/mg_pagerank_wrapper.pyx [hunk 2]
```diff
@@ -54,26 +54,37 @@ def mg_pagerank(input_df, local_data, rank, handle, alpha=0.85, max_iter=100, to
 
     cdef uintptr_t c_identifier = df['vertex'].__cuda_array_interface__['data'][0];
     cdef uintptr_t c_pagerank_val = df['pagerank'].__cuda_array_interface__['data'][0];
-
+    
+    cdef uintptr_t c_pers_vtx = <uintptr_t>NULL
+    cdef uintptr_t c_pers_val = <uintptr_t>NULL
+    cdef sz = 0
```
Comment (kaatish) [inline python/cugraph/opg/link_analysis/mg_pagerank_wrapper.pyx:60]: I think sz needs a type if I am not mistaken.

File: cpp/src/link_analysis/pagerank.cu (hunk 1 above)
Comment (afender) [reply cpp/src/link_analysis/pagerank.cu]: I updated the whole codebase to remove `opg` occurrences

File: python/cugraph/opg/link_analysis/mg_pagerank_wrapper.pyx (hunk 2 above)
Comment (afender) [reply python/cugraph/opg/link_analysis/mg_pagerank_wrapper.pyx]: done

**Comment by afender** [comment]
rerun tests

**Comment by afender** [comment]
rerun tests

**Comment by afender** [comment]
rerun tests

**Comment by BradReesWork** [comment]
rerun tests

**Pull request merged**